rpc/calibration/RpcCal2Hit.C

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
Calibration macros should be in https://github.com/R3BRootGroup/R3BParams_S522

Reply:
These types of files have been moved, sorry!